test/functional/settings_test.rb

```diff
@@ -0,0 +1,39 @@
+require File.join(File.dirname(__FILE__), '..', 'test_helper')
+
+describe 'Settings' do
+  let(:setting) do
+  {
+    :name => 'content_action_accept_timeout',
+    :full_name => 'Accept action timeout',
+    :value => 20,
+    :description => 'Time in seconds to wait for a Host to pickup a remote action'
+
+  }
+  end
+
+  describe 'list command' do
+    before do
+      @cmd = %w(settings list)
+    end
+
+    params = []
```

Comment:
I think we can remove the empty params in this case

Reply:
Hi @tstrachota , I have made the changes as per suggestions. In case of params, if I remove the empty params, it then gives an error as it is not able to find 'params' later in the file. Hence, kept it as is.